src/components/pages/Login/LoginContainer.js

```diff
@@ -62,9 +62,12 @@ const LoginContainer = () => {
         </div>
         <div className="login-page">
           <div id="sign-in-widget" aria-label="login form" />
-          <Link className="link-styles" to="/signup">
-            <p>Don't have an account? Register here</p>
-          </Link>
+          <p className="register">
+            Don't have an account?{' '}
+            <Link className="link-styles" to="/signup">
+              <span>Register here</span>
+            </Link>
```

Comment:
Very nice, I like the way our link is separated to include just the 'Register Here' call to action - this'll look even nicer than what we had!